### PR TITLE
Update acceptance tests for introduction of connection menus

### PR DESCRIPTION
**Pull request by njseeto**
[Trello](https://trello.com/c/pnEnLA6l/2383-update-acceptance-tests-for-long-lived-branch-m)

### Up wait time
This is the maximum amount of time capybara will wait for a test.

### Remove no branching spec references to it in circle config
We no longer require this spec as we will not need the feature flag anymore.

### Increase parallelism to 7

Co-authored-by: Chris Pymm <chris.pymm@digital.justice.gov.uk>

### CODE REVIEW

File: acceptance/support/capybara.rb [hunk 1]
```diff
@@ -11,4 +11,4 @@
 Capybara.default_host = ENV['ACCEPTANCE_TESTS_EDITOR_APP']
 Capybara.app_host = ENV['ACCEPTANCE_TESTS_EDITOR_APP']
 Capybara.run_server = false
-Capybara.default_max_wait_time = 5
+Capybara.default_max_wait_time = 15
```
Comment (brenetic) [inline acceptance/support/capybara.rb:14]: wow

File: acceptance/support/common_steps.rb [hunk 2]
```diff
@@ -17,28 +17,28 @@ def given_I_am_logged_in
...
-      editor.execute_script(
-        "document.getElementById('password').value = '#{ENV['ACCEPTANCE_TESTS_PASSWORD']}'"
-      )
-      editor.execute_script(
-        "document.getElementById('btn-login').click()"
-      )
-    else
-      editor.sign_in_email_field.set('form-builder-developers@digital.justice.gov.uk')
-      editor.sign_in_submit.click
+    using_wait_time 10 do
```
Comment (brenetic) [inline acceptance/support/common_steps.rb:20]: if default wait time has gone up to 15 then maybe this is no longer needed?

Comment (njseeto) [reply acceptance/support/common_steps.rb:20]: Good point. I've changed this and am running tests locally, which are passing.

**Review by brenetic — APPROVED** [review]
🚀  awesome stuff!

**Pull request merged**